client/src/types.d.ts

```diff
@@ -5,7 +5,11 @@ import type {
   LanguageClientOptions,
 } from "vscode-languageclient/node";
```

Comment:
> This will require a similar feature in the Deno language server to only provide language service features for the enabled paths.

So this change doesn't affect the CLI doing diagnostics for a non-enabled folder, correct? As a temporary solution, is there a way we could use the client to not send files outside that enabled folder unless they're imported?

Reply:
> As a temporary solution, is there a way we could use the client to not send files outside that enabled folder unless they're imported? (Sorry, I could research this myself, but it's late here).

No, it gets even worse, you can't really do that.  I am working on the lsp implementation.